### PR TITLE
Delete exceptions from homebrew-fork

### PR DESCRIPTION
**Pull request by AnastasiaSulyagina**
### Changes to the core
- [x] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

Using exceptions from homebrew now as a preparation to core merge([#14384](https://github.com/caskroom/homebrew-cask/issues/14384)).

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
👏 It's also worth noting that when https://github.com/Homebrew/brew/pull/390 is merged we'll be verifying that Homebrew's core changes don't end up breaking `brew cask` which means it'll be safer to rely on Homebrew's internals.

File: Rakefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ require 'rake/testtask'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
 
+$LOAD_PATH.unshift(File.expand_path('../../../../Homebrew', __FILE__))
```
Comment (jawshooah) [inline Rakefile]: There's no guarantee that tests are being run from the tap repo. For example, I have a separate clone in a common directory for Git projects where I do most of my development, so this wouldn't work for me.

Better, I think, to do this:

```ruby
homebrew_repo = `brew --repository`
$LOAD_PATH.unshift(File.expand_path("#{homebrew_repo}/Library/Homebrew"))
```

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -2,6 +2,9 @@
 require 'rspec/its'
 require 'rspec/wait'
 
+homebrew_library_path = Pathname.new(File.expand_path(__FILE__+'/../../../../../Homebrew'))
```
Comment (jawshooah) [inline spec/spec_helper.rb]: Same problem here

File: test/test_helper.rb [hunk 3]
```diff
@@ -2,6 +2,9 @@
 require 'bundler/setup'
 require 'pathname'
 
+homebrew_library_path = Pathname.new(File.expand_path(__FILE__+'/../../../../../Homebrew'))
```
Comment (jawshooah) [inline test/test_helper.rb]: And here

**Comment by jawshooah** [comment]
Great work, thanks!

File: Rakefile (hunk 1 above)
Comment (luchiano1) [inline Rakefile]: 1

File: spec/spec_helper.rb (hunk 2 above)
Comment (luchiano1) [inline spec/spec_helper.rb]: 1

File: test/test_helper.rb (hunk 3 above)
Comment (luchiano1) [inline test/test_helper.rb]: 1

**Comment by luchiano1** [comment]
1

**Pull request merged**